### PR TITLE
update logging system to log slash commands

### LINKED ISSUE #53: Improve: Logging system
Currently, the logging system logs every command that is sent to the bot using the prefix. The logging system does not log any slash commands used. Log the slash commands as shown below:

image

### PR DESCRIPTION
**Pull request by SeethaLakshmiGanesan**
Improved the logging system to log slash commands and add prefix to the log based on the command (either bot prefix or slash).

Slash command log:
image

Command with bot prefix:
image

Closes #53

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -161,6 +162,29 @@ func main() {
...
+	if logChannelID != "" {
+		guild, err := s.Guild(gID)
+		if err != nil {
+			fmt.Printf("Error getting guild info: %s", err)
+			return
+		}
+
+		s.ChannelMessageSend(
+			logChannelID,
+			fmt.Sprintf(`[%s] %s > %s > %s`, prefix, guild.Name, user, command),
```
Comment (z404) [inline main.go:183]: This looks great, making a reusable function was a good idea. A small change here would be nice though.

Currently, the bot prints the following:
`[Slash] Warfstache's server > Wilford Warfstache > time`

I'd like it to be a _little_ more specific by printing this instead:
`[Slash] Warfstache's server > Wilford Warfstache#0256 > time`

That way, we can make sure to narrow a log down to the exact user who used the command.

**Review by z404 — CHANGES_REQUESTED** [review]

**Comment by z404** [comment]
@SeethaLakshmiGanesan Thank you for your contribution! I've requested a tiny change. Also, your branch seems to be 2 commits behind the current branch, make sure to bring it up to date so as to avoid loss of code! Good work!

File: main.go (hunk 1 above)
Comment (SeethaLakshmiGanesan) [reply main.go:183]: Sure, will change it.

**Review by z404 — APPROVED** [review]
LGTM!

**Comment by z404** [comment]
You still have to bring your branch up to date before I merge. Look at [this](https://medium.com/@topspinj/how-to-git-rebase-into-a-forked-repo-c9f05e821c8a) guide on how to achieve it. You could also do it through the GitHub website. You will know you did it correctly when your code has the `guildMemberLeave` function.

**Review by z404 — APPROVED** [review]
Good job!

**Pull request merged**